### ISSUE TITLE
pip installation in isolated mode causes build errors

### ISSUE THREAD

**Issue by ghost** [TkinterEP/python-tksvg#3]
- macOS 12.6.6
- Python 3.11.3
- pip 23.1.2

Running `pip3 install tksvg` (without an environment) gives me this:
```
Collecting tksvg
  Using cached tksvg-0.7.4.tar.gz (50 kB)
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error

  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [14 lines of output]
      Installing dependency cmake... Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "/private/var/folders/9m/b3t01c7x13sfcnk6rqs5pr2c0000gp/T/pip-install-wpxcvtls/tksvg_f11d40390e6440e8a94f27ed11b96779/setup.py", line 62, in <module>
          sp.call(["pacman", "--needed", "--noconfirm", "-S", "mingw-w64-x86_64-{}".format(dep)])  # , stdout=sp.PIPE)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 389, in call
          with Popen(*popenargs, **kwargs) as p:
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 1024, in __init__
          self._execute_child(args, executable, preexec_fn, close_fds,
        File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 1917, in _execute_child
          raise child_exception_type(errno_num, err_msg, err_filename)
      FileNotFoundError: [Errno 2] No such file or directory: 'pacman'
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
```

**Comment by avalon60** [comment]
Same here:
Linux Mint 20.1
Python 3.10

**Comment by RedFantom** [comment]
Thank you for posting and also verifying that [the proposed work-around](https://github.com/israel-dryer/TkFontAwesome/issues/7#issuecomment-1597805936) fixes the issue  🙂 .

**Comment by dishb** [comment]
@RedFantom I'm actually yet to try that solution, completely forgot! 😦, I'll get back to you if it works.

**Comment by dishb** [comment]
Just tried out the workaround, it doesn't work on my end.

**Comment by mizraith** [comment]
Note -- if you can ditch scikit-build-core you will save from bringing in a lot of unnecessary dependencies.   Recently completed a branch of TkFontAwesome where I used hatch as the build engine and the number of dependencies dropped significantly!

**Comment by rdbende** [comment]
Fixed by #6
